test/index_test.js

```diff
@@ -160,7 +161,10 @@ suite('Indexing', () => {
     result = await helper.index.findTask(myns + '.my-task2');
     assert(result.taskId === taskId2, 'Wrong taskId');
     
-    await helper.handlers.Namespace.expireEntries('', helper.handlers.IndexedTask);
+    // Set now to one day in the past 
+    var now = taskcluster.fromNow('- 1 day');
```

Comment:
I just want to note here that the `expires` of IndexedTask are not rounded to the date. So currently we would be expiring all tasks that are at least more than a day past their expiry time. This can lead to a situation where the namespace is deleted( expires rounded off to the date ) while the indexed task under it is not . 
We can easily rectify this though by setting `now` to the current date for the indexed tasks.

Reply:
Oh, hm, good thinking!!

We should probably start rounding namespace expirations *up* one day (so they expire up to 24 hours later than they otherwise would) instead of earlier.  For index entries that already exist, the `-1 day` `expirationDelay` should avoid any issues (neither the namespace nor the index will be deleted until 1 day after they otherwise would be; that they then get deleted in an unexpected order is not going to cause any issues).

Can you make one more commit here to do that rounding-up?